packages/next-app/public/tokens/logoUri.ts

```diff
@@ -0,0 +1,23 @@
+interface ITokenLogoUri {
+  [key: string]: string;
+}
+
+export const tokenLogoUri: ITokenLogoUri = {
+  DAI: "https://assets.coingecko.com/coins/images/9956/small/4943.png?1636636734",
+  WETH: "https://assets.coingecko.com/coins/images/2518/small/weth.png?1628852295",
+  BAL: "https://assets.coingecko.com/coins/images/11683/small/Balancer.png?1592792958",
+  USDT: "https://assets.coingecko.com/coins/images/325/small/Tether.png?1668148663",
+  USDC: "https://assets.coingecko.com/coins/images/6319/small/USD_Coin_icon.png?1547042389",
+  WBTC: "https://assets.coingecko.com/coins/images/7598/small/wrapped_bitcoin_wbtc.png?1548822744",
+  FEI: "https://assets.coingecko.com/coins/images/14570/small/ZqsF51Re_400x400.png?1617082206",
+  GNO: "https://assets.coingecko.com/coins/images/662/small/logo_square_simple_300px.png?1609402668",
+  aDAI: "https://assets.coingecko.com/coins/images/14242/small/aDAI.84b6c41f.png?1615528749",
+  "bb-a-USDT":
+    "https://assets.coingecko.com/coins/images/25826/small/0x2bbf681cc4eb09218bee85ea2a5d3d13fa40fc0c.png?1654063979",
+  "bb-a-USDC":
+    "https://assets.coingecko.com/coins/images/25780/small/bb-a-usdc.png?1653735108",
+  "bb-a-DAI":
+    "https://assets.coingecko.com/coins/images/25883/small/bb-a-dai.png?1654347111",
+  miMATIC:
+    "https://assets.coingecko.com/coins/images/15264/small/mimatic-red.png?1620281018",
+};
```

Comment:
`#would` add [a const assertion here](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) instead of defining a type.